src/modules/formatter.rs

```diff
@@ -7,7 +7,7 @@ use std::{io::{BufWriter, Write}, process::{Command, Stdio}};
 #[derive(Debug, Clone, Copy)]
 #[allow(non_camel_case_types)]
 pub enum BashFormatter {
-    /// https://github.com/mvdan/sh
+    /// <https://github.com/mvdan/sh>
```

Comment:
Change was suggested by RustRover.  Not sure if it's sensible; perhaps someone who knows more about doc comments than I do could verify?

Reply:
As per [rust book](https://rustc-dev-guide.rust-lang.org/rustdoc-internals.html#passes-anything-but-a-gas-station-or-hot-potato)

> bare_urls detects links that are not linkified, e.g., in Markdown such as Go to https://example.com/. It suggests wrapping the link with angle brackets: Go to <https://example.com/>. to linkify it. This is the code behind the rustdoc::bare_urls lint.